rpcs3/Emu/Cell/Modules/libsynth2.cpp

```diff
@@ -3,6 +3,9 @@
 
 #include "libsynth2.h"
 
+#ifdef s_addr
+#undef s_addr
+#endif
```

Comment:
why

Reply:
I will revert, there was a compilation error, but I can not reproduce it anymore.